<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>946</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Product</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>50</number>
      </property>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="cancel_add_edit_product_button">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: red;
	color: white;
	font-size: 20px;
	font-weight:bold;

}</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="submit_add_edit_product_button">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: green;
	color: white;
	font-size: 20px;
	font-weight:bold;

}</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="topMargin">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="remarks_label">
            <property name="text">
             <string>Remarks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="remarks_error_message">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Error Message</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="remarks_input"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>118</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="sku_label">
            <property name="text">
             <string>SKU</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sku_error_message">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Error Message</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="sku_input"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="topMargin">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="cost_price_label">
            <property name="text">
             <string>Cost Price</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="cost_price_error_message">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Error Message</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSpinBox" name="cost_price_input">
          <property name="suffix">
           <string/>
          </property>
          <property name="prefix">
           <string>Rp. </string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="topMargin">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="price_label">
            <property name="text">
             <string>Price</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="price_error_message">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Error Message</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSpinBox" name="price_input">
          <property name="prefix">
           <string>Rp. </string>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="topMargin">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="product_name_label">
            <property name="text">
             <string>Product Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="product_name_error_message">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Error Message</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="product_name_input"/>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="topMargin">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="stock_label">
            <property name="text">
             <string>Stock</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="stock_error_message">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Error Message</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSpinBox" name="stock_input">
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
